java/configuration/src/main/java/sleeper/configuration/properties/UserDefinedInstanceProperty.java

```diff
@@ -642,29 +637,31 @@ public interface UserDefinedInstanceProperty extends InstanceProperty {
             .defaultValue(DEFAULT_BULK_IMPORT_EMR_RELEASE_LABEL.getDefaultValue())
             .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
             .runCDKDeployWhenChanged(true).build();
-    UserDefinedInstanceProperty BULK_IMPORT_PERSISTENT_EMR_MASTER_INSTANCE_TYPES = Index.propertyBuilder("sleeper.bulk.import.persistent.emr.master.instance.types")
-            .description("(Persistent EMR mode only) The EC2 instance types used for the master node of the " +
-                    "persistent EMR cluster. Multiple instance types can be specified separated by commas. One will " +
-                    "be chosen depending on the capacity available.")
-            .defaultValue(DEFAULT_BULK_IMPORT_EMR_MASTER_INSTANCE_TYPES.getDefaultValue())
+    UserDefinedInstanceProperty BULK_IMPORT_PERSISTENT_EMR_MASTER_X86_INSTANCE_TYPES = Index.propertyBuilder("sleeper.bulk.import.persistent.emr.master.x86.instance.types")
+            .description("(Persistent EMR mode only) The EC2 x86 instance types used for the master node of the " +
+                    "persistent EMR cluster. " +
+                    "For more information, see the Bulk import using EMR - Instance types section in docs/05-ingest.md")
+            .defaultValue(DEFAULT_BULK_IMPORT_EMR_MASTER_X86_INSTANCE_TYPES.getDefaultValue())
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
+            .runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_PERSISTENT_EMR_EXECUTOR_X86_INSTANCE_TYPES = Index.propertyBuilder("sleeper.bulk.import.persistent.emr.executor.x86.instance.types")
+            .description("(Persistent EMR mode only) The EC2 x86 instance types used for the executor nodes of the " +
+                    "persistent EMR cluster. " +
+                    "For more information, see the Bulk import using EMR - Instance types section in docs/05-ingest.md")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
+            .runCDKDeployWhenChanged(true).build();
```

Comment:
This property has lost its default value. It was previously set from `DEFAULT_BULK_IMPORT_EMR_MASTER_INSTANCE_TYPES.getDefaultValue()`.